src/components/JaegerIntegration/TracesComponent.tsx

```diff
@@ -41,6 +38,7 @@ interface TracesProps {
   namespaceSelector: boolean;
   showErrors: boolean;
   duration: number;
+  lastRefresh: number;
```

Comment:
These should likely both be typed using out existing definitions, like DurationinSeconds and TimeInMilliseconds (those are examples, not sure if those units are correct).  This comment can likely be applied in multiple places.

Reply:
Good point, I didn't see that this property was injected from the refresh container.
Then perhaps it's not necessary in children components.
I need to work on it.